### DISCUSSION TITLE
Can I open PDN files?

### DISCUSSION THREAD

**Discussion by AshleyPikachu** [PintaProject/Pinta#827] [Q&A] [answered]
Trying to get off Paint.net, title pretty much sums it up.
Also, how do I save a work file like a .pdn to save layered work to come back to?

 Thanks!

**Comment by cameronwhite** [comment, marked as answer]
For `.pdn` files, the answer is complicated :) There was an add-in that previously did this (https://github.com/stevekew/PaintDotNetAddin), but support for add-ins was dropped in Pinta 2.0 for various technical reasons. But add-in support will be returning in Pinta 2.2 so that add-in will work again in the future after some updates

The equivalent to a `.pdn` file would be saving to the `.ora` format (https://en.wikipedia.org/wiki/OpenRaster)

**Reply by AshleyPikachu** [reply]
Thanks for the heads up about the plugins and ora! :)
